app/helpers/application_helper.rb

```diff
@@ -57,7 +57,7 @@ def autolink(link, link_type)
     end
 
     def url_link(link)
-      if URI.parse(link).host == "www.youtube.com"
+      if URI.parse(link).host == "www.youtube.com" && link.index('v=')
```

Comment:
I think this isnt the best idea. we should look at youtube video regex instead because people could by accidentally crash the site by just doing `www.youtube.com/v=` which is an invalid URL

Reply:
So what should that regex check? `www.youtube.com/*/v=`?

Reply:
i think that regex can lead to other issues like `www.youtube.com//v=` which again isnt valid. let me look into it and suggest a regex